src/pages/roadmap.mdx

```diff
@@ -22,7 +22,7 @@ Stork is a single-person project that I build in my spare time. Development will
 ## Large Features
 
 - Support fuzzy search
-- Create and read sharded indexes
+- Create and read shared indexes
```

Comment:
This one actually isn't a typo! I'm describing [a single search index split up into different files](https://en.wikipedia.org/wiki/Shard_(database_architecture)) here.

Reply:
Thanks for clarification! I reverted this meanwhile.